inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```

Comment:
```suggestion
    mmm = np.mean(data, axis=0)
```
Instead of `mmm` it is better to use like `data_mean` or something similar